### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by vinayrk**
solution to assignment 1

### CODE REVIEW

File: CP/Assignment1-STL/Vector.cpp [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            cin >> x >> y;
+            v.at(x) = y;
+        }
+        else if (c == 3)
+        {
+            f = 0;
+            cin >> x;
+            for (i = v.begin(); i <= v.end(); i++)
+            {
+                if (*i == x)
```
Comment (Aj163) [inline CP/Assignment1-STL/Vector.cpp]: Use find()

File: CP/Assignment1-STL/Vector.cpp [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                cout << "YES" << endl;
+            else
+                cout << "NO" << endl;
+        }
+        else if (c == 4)
+        {
+            cin >> x;
+            for (i = v.begin(); i <= v.end(); i++)
+            {
+                if (*i == x)
```
Comment (Aj163) [inline CP/Assignment1-STL/Vector.cpp]: Use find()

File: CP/Assignment1-STL/array.cpp [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        {
+            cin >> x >> y;
+            a.at(x) = y;
+        }
+        else if (c == 2)
+            sort(a.begin(), a.end());
+        else if (c == 3)
+        {
+            cin >> x;
+            for (j = 0; j < 1000; j++)
```
Comment (Aj163) [inline CP/Assignment1-STL/array.cpp]: Use lowerbound()

File: CP/Assignment1-STL/array.cpp [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+                    break;
+            }
+            cout << j << endl;
+        }
+        else if (c == 4)
+        {
+            cin >> x;
+            for (j = 999; j >= 0; j--)
+            {
+                if (a.at(j) == x)
```
Comment (Aj163) [inline CP/Assignment1-STL/array.cpp]: Use upperbound()

File: CP/Assignment1-STL/map.cpp [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+        else if (c == 2)
+        {
+            cin >> x;
+            itr = m.find(x);
+            cout << itr->second << endl;
+        }
+        else if (c == 3)
+        {
+            cin >> x;
+            if (m.count(x) > 0)
```
Comment (Aj163) [inline CP/Assignment1-STL/map.cpp]: Use find()

File: CP/Assignment1-STL/pair.cpp [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+
+        sort(a.begin(), a.end());
+      }
+
+      else
+
+          if (x == 2)
+
+        sort(a.begin(), a.end(), [](const pair<int, int> &lhs, const pair<int, int> &rhs) {
+          return lhs.second < rhs.second;
```
Comment (Aj163) [inline CP/Assignment1-STL/pair.cpp]: If second values are equal, check for first values

File: CP/Assignment1-STL/set.cpp [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+            cout << endl;
+        }
+        else if (c == 6)
+        {
+            cout << *m.end() << endl;
+        }
+        else if (c == 7)
+        {
+            cin >> x;
+            if (m.count(x) > 0)
```
Comment (Aj163) [inline CP/Assignment1-STL/set.cpp]: Use find()

**Comment by vinayrk** [comment]
It is in my folder only

**Pull request merged**